### PR TITLE
typos

### PR DESCRIPTION
**Pull request by ralphtheninja**

### CODE REVIEW

**Comment by ralphtheninja** [comment]
I found some other typos as well, but I can update with new PRs for those changes.

**Comment by emqplus** [comment]
@ralphtheninja, Thx.

File: docs/source/config.rst [hunk 1]
```diff
@@ -541,7 +541,7 @@ Plugins Folder
 TCP Listeners
 -------------
 
-Congfigure the TCP listeners for MQTT, MQTT(SSL) and HTTP Protocols.
+Configure the TCP listeners for MQTT, MQTT(SSL) and HTTP Protocols.
 
 The most important parameter is 'max_clients' - max concurrent clients allowed.
```
Comment (ralphtheninja) [inline docs/source/config.rst]: @emqplus Have you considered using back tics instead of simple quotes to emphasize words more? E.g. like `max_clients`

**Comment by emqplus** [comment]
@ralphtheninja, It's better to use back tics, I think. I will update the docs later:)

**Pull request merged**